config/sprayproxy.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright © 2023 The Spray Proxy Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sprayproxy
+  namespace: redhat-sprayproxy
+  labels:
+    app.kubernetes.io/name: sprayproxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: sprayproxy
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: sprayproxy
+    spec:
+      serviceAccountName: redhat-sprayproxy
+      containers:
+        - name: sprayproxy
```

Comment:
Other things needed in the container spec:

- Port declarations for http (8080) and metrics (9090 I believe, check the code to verify the default value).

Reply:
Added, it is 6000